### DISCUSSION TITLE
FNIRSI GC-01: Flash instructions

### DISCUSSION THREAD

**Discussion by jackhen** [Gissio/radpro#44] [Show and tell]
Ok Guys, here is the foolproff to get the FW loaded. Works everytime.....
After you connect your GC-01 via usb to your PC you "hold down" the power switch. As long as you hold it down, you will see the disk appear on your PC. So hold it down, then drag the repo onto your disk icon. Release the power button when you see the GC-01 booting. Takes only a few seconds. And it doesn't matter the USB protocol. It just works.

**Comment by Gissio** [comment]
Does it work for others, too?

**Reply by PsychoBitchy** [reply]
My device (FNIRSI GC-01 with "GC01 v0.2" written on PCB) acts differently:
With the unit switched off, i can equally put it in disk volume mode by clicking once, then click again to reverse, OR having the OK button pressed and the volume disappears on  release after I uploaded the firmware.
